test/e2e/policy-seq-oauth2-expression-log-ratelimit-proxy.js

```diff
@@ -39,7 +39,7 @@ describe('E2E: oauth2, proxy, log, expression, rate-limit policies', () => {
           scopes: ['authorizedScope']
         }
       },
-      policies: ['oauth2', 'proxy', 'log', 'expression', 'rate-limit'],
+      policies: ['jwt', 'oauth2', 'proxy', 'log', 'expression', 'rate-limit'],
```

Comment:
Same here, why did you need to add this policy?

Reply:
For this one, with debug
```
  605 passing (53s)
  2 pending
  2 failing

  1) E2E: oauth2, proxy, log, expression, rate-limit policies
       "before all" hook: setup:
     can't resolve reference jwt.json from id http://express-gateway.io/schemas/policies/oauth2.json#
  

  2) E2E: oauth2, proxy, log, expression, rate-limit policies
       "after all" hook: cleanup:
     TypeError: Cannot read property 'close' of undefined
      at Context.after (test/e2e/policy-seq-oauth2-expression-lo
```
So added `jwt` policy